### PR TITLE
Break out Community section on Learn page

### PR DESCRIPTION
**Pull request by paf31**
Work towards #50.

### CODE REVIEW

**Comment by paf31** [comment]
Ping.

File: learn/index.html [hunk 1]
```diff
@@ -33,23 +33,39 @@
 
     <section>
 
-      <h2>Learning resources</h2>
+      <h2>Learning Resources</h2>
```
Comment (garyb) [inline learn/index.html]: We use sentence case for the other section headers, so this was right before

File: learn/index.html [hunk 2]
```diff
@@ -58,6 +74,13 @@
           </div>
         </li>
         <li>
+          <h3>GitHub Wiki</h3>
+          <p>The wiki is the primary source for information on the language and compiler.</p>
```
Comment (garyb) [inline learn/index.html]: Is this description accurate still? :smile:

**Comment by paf31** [comment]
Closing, this needs to be reimplemented using the new Hakyll setup.

**Pull request closed without merging**